File: UMA.App.IdentityManager/Role/Queries/GetRoleQuery.cs
using MediatR;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using UMA.App.IdentityManager.Role.Models;

namespace UMA.App.IdentityManager.Role.Queries
{
    public class GetRoleQuery : IRequest<RoleViewModel>
    {
        public Guid? Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
    }
}

File: UMA.Web.Angular/UMA-Web-Angular/src/app/_entities/role/query/getRole.query.ts
import { IRequest } from 'src/app/_models/interfaces/IRequest';
import { RoleViewModel } from '../reponse/role.viewmodel';

export class GetRoleQuery implements IRequest<RoleViewModel> {
  _response?: RoleViewModel;
  id?: string;
  name?: string;
  description?: string;
}
